src/Modifier/RemoveGroupsFromVendor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Config\Modifier;
+
+use function is_array;
+
+/**
+ * @see RemoveFromVendor::groups()
+ */
+final class RemoveGroupsFromVendor
```

Comment:
Maybe mark as internal?

Reply:
`RemoveFromVendor::groups()` returned this to userland... I do not know if it is necessary to do internal.

Reply:
OK